ios/RNSScreenStack.m

```diff
@@ -17,6 +18,28 @@ @interface RNSScreenStackView () <UINavigationControllerDelegate, UIAdaptivePres
 
 @end
 
+@implementation RNScreensNavigationController
+
+- (UIStatusBarStyle)preferredStatusBarStyle
+{
+  UIViewController *childVC =  [[self childViewControllers] lastObject];
+  return [childVC isKindOfClass:[RNSScreen class]] ? childVC.preferredStatusBarStyle : UIStatusBarStyleDefault;
+}
+
+- (UIStatusBarAnimation)preferredStatusBarUpdateAnimation
+{
+  return [[self childViewControllers] lastObject].preferredStatusBarUpdateAnimation;
```

Comment:
maybe we should use `topViewController` instead of `childViewControllers lastObject` here and in the previous method?